Project page docs

aryand10 opened the pull request:
Complete documentation and adding comments for all files related to Projects Page to adhere to JSDocs formatting to resolve #38

aryand10 committed:
JSDocs documentation and comments for JS files

Added JSDocs formatted headers and other helpful comments for clarity and documentation efforts for projects page

aryand10 committed:
Added comments for all other files

Added comments for clarity for non JS files for Projects Page

schowbey reviewed (commented):
Looks good. Please remove the duplicate google fonts link. Also, I don't think you need the import font statement in CSS. All the other pages have a favicon except yours, but I'll fix that in my styling branch.

aryand10 committed:
Duplicate link removed for html and font url removed from CSS

As requested in PR review, the duplicate link for fonts was removed from the HTML and the import font statement was removed from the css to facilitate #41

schowbey reviewed (approved):
all good! thanks
